test/parallel/test-repl.js

```diff
@@ -267,8 +268,8 @@ function error_test() {
       expect: /\bSyntaxError: Invalid or unexpected token/ },
     // do not fail when a String is created with line continuation
     { client: client_unix, send: '\'the\\\nfourth\\\neye\'',
-      expect: `${prompt_multiline}${prompt_multiline}'thefourtheye'\n${
-              prompt_unix}` },
+      expect: `${prompt_multiline}${prompt_multiline}'thefourtheye'\n` +
```

Comment:
Splitting at `\n` made sense elsewhere as it made the template literal layout approximate its output, although I can see that maybe it's not needed here.